modules/govuk_jenkins/templates/jobs/run_metadata_tagger.yaml.erb

```diff
@@ -0,0 +1,24 @@
+---
+- job:
+    name: Run_Metadata_Tagger
```

Comment:
I wonder if should call this `Run Rummager/Search Metadata Tagger` just in case it's unclear what it's for.